test/powershell/engine/Remoting/RemoteSession.Basic.Tests.ps1

```diff
@@ -1,11 +1,14 @@
-
 Describe "New-PSSession basic test" -Tag @("CI") {
+    if ( ! $IsWindows ) {
```

Comment:
Please use [current test guidance](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md#skipping-tests-in-bulk) to skip tests.